### ISSUE TITLE
Tammann notebook issues

### ISSUE THREAD

**Issue by ketch** [clawpack/riemann_book#93]
I noticed a few issues that I wasn't able to deal with myself:

- It might be convenient to just work out everything for either the 1-wave and then say "similarly, for the 3-wave we obtain..." instead of keeping the parameters $k$ and $n$ everywhere.
- In 6.1.2: "As the entropy is invariant" -- with respect to what?
- Also in 6.1.2, there is a sentence "for numerical purposes" that seems a bit out of place since there are no other comments about numerical methods here.
- Near the end of 6.1.2 instead of saying that the function $\Phi$ "might change in each iteration of the root finder", I think it would be nice (as we do elsewhere) to just define the function piecewise.
- At the start of 6.1.4, the first equation reference: I'm not sure this is the equation we mean to refer to.
- There is a reference near the beginning of 6.2.1 that is not defined in our bibliography.
- The script F notation seems unnecessary -- can we dispense with it?

**Comment by maojrs** [comment]
- I personally do not mind having $k$ and $n$, but we can discuss this.
- I wrote: 'As the entropy is invariant along the rarefaction wave' instead.
- I removed the whole sentence: 'For numerical purposes, a simple approximate velocity is provided for  s1s1  and  s3s3  as the average between the head and tail velocity.'
- I added the cases for $\Phi$, although it was already stated in the beginning.
- I believe that the eq. reference in 6.1.4 is correct
- I added the missing reference to riemann.bib
- I think it helps keeping track of the calculation to have the F, though it is a matter of style, so we can go either way.

I am going do a pull request with these changes to ketch:revise_later_chapters, so they can be incorporated with your latest pull request.
